<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TypeCheck</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/new/prefix1/img/icon.jpg</normaloff>:/new/prefix1/img/icon.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Time:</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdTimer">
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="digitCount">
         <number>4</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Total symbols:</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdSymbols">
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Avg.:</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdAvg">
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Set limit(sec):</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleLimit">
        <property name="maximum">
         <double>9999.989999999999782</double>
        </property>
        <property name="value">
         <double>60.000000000000000</double>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="btnReset">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>239</red>
             <green>41</green>
             <blue>41</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>239</red>
             <green>41</green>
             <blue>41</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>239</red>
             <green>41</green>
             <blue>41</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="text">
        <string>Reset Avg.</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_14">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="btnRun">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>138</red>
              <green>226</green>
              <blue>52</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>138</red>
              <green>226</green>
              <blue>52</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>138</red>
              <green>226</green>
              <blue>52</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="text">
         <string>Run</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+R</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btnAbort">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>239</red>
              <green>41</green>
              <blue>41</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>239</red>
              <green>41</green>
              <blue>41</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>239</red>
              <green>41</green>
              <blue>41</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Abort</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSplitter" name="splitter_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QComboBox" name="KbdLayout">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>En</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ru</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Jpn</string>
          </property>
         </item>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textInput">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>220</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>220</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>239</red>
            <green>239</green>
            <blue>239</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="documentTitle">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_13">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_11">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPlainTextEdit" name="textPreview"/>
       <widget class="QSplitter" name="splitter_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Dictionary build time(sec):</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdDict"/>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_12">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Spelling mistakes</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listErrors"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
